swagger.yaml

```diff
@@ -689,12 +696,12 @@ definitions:
     description: "The group model for the response body"
     type: object
     properties:
+      id:
+        description: "The group id"
+        type: string
       name:
         description: "The group name"
         type: string
-      description:
-        description: "A description of the group"
-        type: string
       precedence:
         description: "The priority of the group"
         type: integer
```

Comment:
The enum on line 799 is not aligned to the list of possible values in [`error_content.go`](https://github.com/ONSdigital/dp-identity-api/blob/feature/rename-group-entity-fields/models/error_content.go#L10-L46).

Reply:
Im not sure what you mean by this comment.

Reply:
Sorry, that wasn't very clear. The list of possible enum values for the enum on line 799 is incorrect and does not match the list of possible values which can be found in the code linked to in the comment above.